apiserver/facades/controller/firewaller/firewaller.go

```diff
@@ -86,14 +87,15 @@ func NewStateFirewallerAPI(
 	modelConfigService ModelConfigService,
 	applicationService ApplicationService,
 	machineService MachineService,
+	controllerService ControllerService,
+	modelUUID model.UUID,
 	logger corelogger.Logger,
 ) (*FirewallerAPI, error) {
 	if !authorizer.AuthController() {
 		// Firewaller must run as a controller.
 		return nil, apiservererrors.ErrPerm
 	}
 	// Set up the various authorization checkers.
-	accessModel := common.AuthFuncForTagKind(names.ModelTagKind)
 	accessUnit := common.AuthFuncForTagKind(names.UnitTagKind)
 	accessApplication := common.AuthFuncForTagKind(names.ApplicationTagKind)
```

Comment:
I think this can be deleted as well. It is only used in `GetExposeInfo` in the API. This facade is never called by the client.

Recap:
- Remove accessApplication
- Remove `GetExposeInfo` from the api.
- Delete the funcs from the client as well (https://github.com/tlm/juju/blob/f9bb18a7a3e85048c1c7aec60b0312196302643c/api/controller/firewaller/application.go#L34)

Reply:
Sure will remove this, thanks!